test/pages/AddRepresentativeYesNoPageSpec.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import pages.behaviours.PageBehaviours
+
+class AddRepresentativeYesNoPageSpec extends PageBehaviours {
+
+  "AddRepresentativeYesNoPage" - {
+
+    beRetrievable[Boolean](AddRepresentativeYesNoPage)
+
+    beSettable[Boolean](AddRepresentativeYesNoPage)
+
+    beRemovable[Boolean](AddRepresentativeYesNoPage)
+
+//    "cleanup" - {}
```

Comment:
Probably worth either removing this line or adding a TODO: so that it is not missed.